string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,35 @@
+/* exported isVowel */
+// Check if char is a string or if the length of char is greater than 2,
+// If it is not a string or if the length of char is greater than 2, return 'Invalid string'
+// If it is a string,
+// Check if the character is 'a'
+// if true, return 'true'
+// Check if the character is 'e'
+// if true, return 'true'
+// Check if the character is 'i'
+// if true, return 'true'
+// Check if the character is 'o'
+// if true, return 'true'
+// Check if the character is 'u'
+// if true, return 'true'
+// Check if the character is 'A'
+// if true, return 'true'
+// Check if the character is 'E'
+// if true, return 'true'
+// Check if the character is 'I'
+// if true, return 'true'
+// Check if the character is 'O'
+// if true, return 'true'
+// Check if the character is 'U'
+// if true, return 'true'
+// If character does not match any of the conditions above
+// then return 'false'
+function isVowel(char) {
+  if (typeof char !== 'string' || char.length > 2) {
+    return 'Invalid string';
+  } else if (char === 'a' || char === 'A' || char === 'e' || char === 'E' || char === 'i' || char === 'I' || char === 'o' || char === 'O' || char === 'u' || char === 'U') {
```

Comment:
Your fingers must be tired from all that typing 😅 

Extra points: Can you think of a way to refactor this? 

There are a few ways to make this a little easier...
- What string methods could you use to avoid checking capital AND lowercase values? 
- This could also be a great candidate for a `switch` statement - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch
- If not using a `switch`, could you keep all the vowels in different data type? What data type could be useful for this "list" of characters? What methods on that "list" could you use to compare against?